docs/Developers/Git_Basics.rst

```diff
@@ -70,7 +70,10 @@ Creating a new branch
 ---------------------
 
 To start working on an issue, you first need to create a new branch where you
-will work.
+will work. You should always have a clean and working branch which
```

Comment:
The sentence can be shortened to `You should always have a clean and working "master" branch`